src/main/kotlin/DetektDemo.kt

```diff
@@ -1,2 +1,55 @@
 class DetektDemo {
-}
\ No newline at end of file
+    private val unusedVariable: String = "unused"
+
+    fun isHTTPSuccess(statusCode: Int): Boolean = statusCode == 200
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/tokgozmusa/detekt-danger/blob/50ce9000fbc0665351fc0cbb11f07ed8a3b0b789/src/main/kotlin/DetektDemo.kt#L4"></span>This expression contains a magic number. Consider defining it to a well named constant.</td>
    </tr>
  </tbody>
</table>